l0/terraform.tfstate.backup

```diff
@@ -0,0 +1,61 @@
+{
+  "version": 4,
+  "terraform_version": "1.1.3",
+  "serial": 41,
+  "lineage": "61117a91-e0ac-48fa-5d88-96a1e7eee56e",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+      "type": "aws_s3_bucket",
+      "name": "storage",
+      "provider": "provider[\"registry.terraform.io/hashicorp/aws\"]",
+      "instances": [
+        {
+          "schema_version": 0,
+          "attributes": {
+            "acceleration_status": "",
+            "acl": "private",
+            "arn": "arn:aws:s3:::my-bucket-for-tf-state",
+            "bucket": "my-bucket-for-tf-state",
+            "bucket_domain_name": "my-bucket-for-tf-state.s3.amazonaws.com",
+            "bucket_prefix": null,
+            "bucket_regional_domain_name": "my-bucket-for-tf-state.s3.us-west-2.amazonaws.com",
+            "cors_rule": [],
+            "force_destroy": false,
+            "grant": [],
+            "hosted_zone_id": "Z3BJ6K6RIION7M",
+            "id": "my-bucket-for-tf-state",
+            "lifecycle_rule": [],
+            "logging": [],
+            "object_lock_configuration": [],
+            "policy": null,
+            "region": "us-west-2",
+            "replication_configuration": [],
+            "request_payer": "BucketOwner",
+            "server_side_encryption_configuration": [],
+            "tags": {
+              "Environment": "Dev",
+              "Name": "my-bucket"
+            },
+            "tags_all": {
+              "Environment": "Dev",
+              "Name": "my-bucket"
+            },
+            "versioning": [
+              {
+                "enabled": true,
+                "mfa_delete": false
+              }
+            ],
+            "website": [],
+            "website_domain": null,
+            "website_endpoint": null
+          },
+          "sensitive_attributes": [],
+          "private": "bnVsbA=="
+        }
+      ]
+    }
+  ]
+}
```

Comment:
Please remove `terraform.tfstate.backup` it's enough that we just have `terraform.tfstate` committed in git

Reply:
Resolved

Reply:
The file is still there. Please check